app/models/exchange_rate_currency_rate.rb

```diff
@@ -12,6 +14,10 @@ def spot_rate?
     [3, 12].include?(validity_start_date.month) && validity_start_date.day == 31 && validity_end_date.nil?
   end
 
+  def countries
```

Comment:
Should this be a has many?

Reply:
@willfish we would need a join table to create a many to many here. I think we can keep it like this or do you recommend anything else?

Reply:
So I figured this is a one to many but without a through? Looks like Sam has this working now?

If we don't use properly defined associations to pull related resources we end up without the ability to eager load stuff. I'll have a proper look tomorrow

Reply:
I haven't had time to review these new changes yet. I will take a look tomorrow.